content/en/os/1.15.x/concepts/shell-less-host/index.markdown

```diff
@@ -26,11 +26,14 @@ The admin container is designed to provide out-of-band access with elevated priv
 On `aws-k8s-*` and `aws-ecs-*` variants, the admin container is not enabled by default but can be turned on or entered through the control container.
```

Comment:
For the lack of a better place for this feedback: there is a type in the commit message and the PR title.

Reply:
😅 Thanks. I should take more time to proof read PR messages.